tsdb/engine/tsm1/encoding.go

```diff
@@ -182,11 +181,7 @@ func (a Values) Deduplicate(ascending bool) Values {
 		other = append(other, val)
 	}
 
-	if ascending {
-		sort.Sort(Values(other))
-	} else {
-		sort.Sort(sort.Reverse(Values(other)))
-	}
+	sort.Sort(Values(other))
```

Comment:
Hmm.  I thought the reason for removing the `ascending` option was to avoid a sort.  In the `desc` cursor implementation, it is still calling `Deduplicate` so it's always sorting anyways.  Removing `ascending` doesn't actually improve performance and just makes the cursor implementation more complicated.  @otoolep thoughts?